tools/parquetGen/main.cpp

```diff
@@ -0,0 +1,1342 @@
+#include <iostream>
```

Comment:
I am not sure what to do with this test data generator.
We obviously won't test regularly so there is high probability the code it will become putrid. I suggest we move the code into the tooling repo.

Reply:
this binary is used within relevant MTR tests